docs/recipes/shark-splash/02-enemies.md

```diff
@@ -110,8 +110,8 @@ game.onUpdateInterval(500, function () {
 
 ## Step 5
 
-Place another ``||Math:pick random 0 to 10||`` block inside the y argument of ``||sprites:set position to||``.
-Change the second argument from 10 to 120 (the height of the screen).
+Get another ``||Math:pick random 0 to 10||`` block and put in inside the ``||sprites:y||`` value of ``||sprites:set position to||``.
+Change the maximum random number from `10` to `120` (the height of the screen).
```

Comment:
the code does 0 to 120,

Reply:
Right, 'maximum' may not be understood as the 'second' number.